src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/copy_data_workspaces.cljs

```diff
@@ -71,6 +86,8 @@
        {:endpoint endpoints/list-workspaces
         :on-done (fn [{:keys [success? status-text get-parsed-response]}]
                    (if success?
-                     (swap! state assoc :workspaces
-                            (remove-self (get-parsed-response) (:workspace-id props)))
+                     (let [all-workspaces (remove-self (:workspace-id props) (get-parsed-response))
+                           filtered-workspaces (filter-workspaces (:this-realm props) all-workspaces)]
+                       (swap! state assoc :workspaces filtered-workspaces
+                         :num-filtered (- (count all-workspaces) (count filtered-workspaces))))
```

Comment:
hmmm. I can't find the bug (and haven't tried too hard), but ... I am consistently seeing "1 workspace unavailable due to conflicting realms" every time I choose to import data. Is there a problem in calculation here? Maybe it has to do with the remove-self?